LibraryApp/src/components/form.js

```diff
@@ -12,36 +13,29 @@ import {
 import {publishers, books} from 'constants/data';
 import {Colors, Global, Spacing, Typography} from 'stylesheets';
 import {validationMessages} from 'constants/locale';
-import {booksList} from 'constants/app-defaults';
-import {setActiveTab, updateBooks} from 'store/actions/library';
+import {booksList} from 'constants/navigators';
+import {updateBooks} from 'store/actions/library';
 import {omit, alert} from 'helpers/application';
 import {GetBookDetails} from 'helpers/library';
 import {isEmail, isFieldEmpty, isUrl} from 'helpers/validation';
 import {InputField, Checkbox, Dropdown} from 'components/shared/form-controls';
-import {previous} from 'constants/icons';
+import {
+  addHardwareBackAction,
+  removeHardwareBackAction,
+} from 'helpers/handle-event-listeners';
+
+const {rowFlex, flexPoint45, spaceBetween, horizontalCenter, center} = Global;
+const {fs20, bold} = Typography;
+const {p10, p20, py10, m10, py20, mtAuto} = Spacing;
+const {lightGreen, white} = Colors;
 
-const {
-  rowFlex,
-  flexPoint45,
-  spaceBetween,
-  horizontalCenter,
-  verticalCenter,
-  center,
-} = Global;
-const {iconFont, fs16, fs20, bold} = Typography;
-const {p10, p15, py10, m10, py20, px15, mtAuto} = Spacing;
-const {blue, lightGreen, white} = Colors;
+const defaultObj = {};
 
 class Form extends React.PureComponent {
   constructor(props) {
     super(props);
-    const {
-      id = '',
-      name = '',
-      author = '',
-      publisher = '',
-      price = '',
-    } = props.selectedBook;
+    const {id = '', name = '', author = '', publisher = '', price = ''} =
```

Comment:
why are we changing this formatting?

Reply:
Just destructuring the selectedBook details to use it in state.